tests/cypress/integration/landingPage.spec.js

```diff
@@ -38,17 +38,30 @@ describe('App', () => {
       cy.get('[data-testid=landing-page-emergency-prompt] summary')
         .click()
         .then(() => {
-          cy.get('[data-testid=landing-page-emergency-info]')
-            .should('be.visible')
+          cy.get('[data-testid=landing-page-emergency-info]').should(
+            'be.visible'
+          );
         });
     });
   });
 
   it('displays a start button', () => {
-    cy.get('a').contains('Start now').should('have.attr', 'href', '/report-repair/priority-list');
+    cy.get('a')
+      .contains('Start now')
+      .should('have.attr', 'href', '/report-repair/priority-list');
   });
 
   it('has an accessibility link', () => {
-    cy.get('a').contains('Accessibility').should('have.attr', 'href', 'https://www.lincoln.gov.uk/accessibility');
```

Comment:
Can this link text be 'Accessibilty statement' for a bit more context. 

Can these links in the footer aslo have a `govuk-footer__link` class attribute so they're styled as such.